packages/neon/neon/lib/src/models/account_cache.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:neon/models.dart';
+import 'package:neon/src/models/disposable.dart';
+
+/// Cache for [Account] specific [Disposable] objects.
+class AccountCache<T extends Disposable> implements Disposable {
+  AccountCache();
+
+  final Map<String, T> _cache = {};
+
+  /// Disposes all entries and clears the cache.
+  @override
+  void dispose() {
+    _cache
+      ..disposeAll()
+      ..clear();
+  }
+
+  /// Updates the cache against the given [accounts].
+  ///
+  /// Every cache entry with an account no longer in the `accounts` is removed
+  /// and disposed. This method is in O(N²).
+  void update(final Iterable<Account> accounts) {
+    _cache.removeWhere((final key, final value) {
+      if (accounts.tryFind(key) == null) {
+        value.dispose();
+        return true;
+      }
+
+      return false;
+    });
+  }
+
+  /// Removes [account] and its associated value, if present, from the cache.
+  ///
+  ///If present the value associated with `account` is disposed.
+  void remove(final Account? account) {
+    final removed = _cache.remove(account?.id);
+    removed?.dispose();
+  }
+
+  /// The value for the given [account], or `null` if [account] is not in the cache.
+  T? operator [](final Account account) => _cache[account.id];
+
+  /// Associates the [account] with the given [value].
+  ///
+  /// If the account was already in the cache, its associated value is changed.
```

Comment:
```suggestion
  /// If the account was already in the cache, it's associated value is changed.
```

Reply:
This is correct.
Its in the form of "sein/seine"
Not it is like "es ist"